python/tests/test_fastdiff.py

```diff
@@ -8,21 +8,21 @@
 
 from fastdiff import compare
 from fastdiff._base import compare as compare_base
-try:
-    from fastdiff._native import compare as compare_native, initiate_instance
-except:
-    def compare_native(a, b):
-        return []
-
-    def initiate_instance():
-        pass
+from fastdiff._native import compare as compare_native, initiate_instance
```

Comment:
This will not allow testing on Python 2.7 (or in a platform where wasmer is not available).

So rather than fail there, I think is better to silently skip it.
```suggestion
try:
    from fastdiff._native import compare as compare_native, initiate_instance
except:
    def compare_native(a, b):
        return []

    def initiate_instance():
        pass
```